### PR TITLE
Allows display manager list for all permissions

### LINKED ISSUE #446: Display group managers for members/basic managers
## Overview

Let group members and managers with low permission list the managers (even if they cannot edit).

## Acceptance criteria

- [ ] Rename the "Administration" tab of groups to "Managers"
- [ ] display the tab for all group members and any level of group managers

### PR DESCRIPTION
**Pull request by Iloveall**
Fixes #446

### CODE REVIEW

**Review by smadbe — CHANGES_REQUESTED** [review]
https://dev.algorea.org/branch/feature/display-group-managers-for-members-or-basic-managers/en/#/groups/by-id/52767158366271444;path=/details -> user is a member of the group (see indication in the blue panel) but cannot see the managers tab.

And where do you implement that non-member cannot see the tab?

Have you tested it completely?

File: src/app/modules/group/pages/group-details/group-details.component.html [hunk 1]
```diff
@@ -10,13 +10,13 @@
...
         class="group-indicator"
-        *ngIf="group?.currentUserManagership !== 'none' || group?.currentUserMembership !== 'none'"
+        *ngIf="group.currentUserManagership !== 'none' || group.currentUserMembership !== 'none'"
         [group]="group"
     >
     </alg-group-indicator>
     <!-- tabs -- if only the first tab is visible, do not show the tab bar -->
-    <div class="nav-tab" *ngIf="group.isCurrentUserManager">
-      <div
+    <div class="nav-tab" *ngIf="group.isCurrentUserManager || group.currentUserManagership !== 'none' || group.currentUserMembership !== 'none'">
```
Comment (smadbe) [inline src/app/modules/group/pages/group-details/group-details.component.html:18]: `group.isCurrentUserManager` covers `group.currentUserManagership !== 'none'` so it is useless here. 
And as usual, you also need to display the tab if the user arrives on the page via a link... in this case, you have to show the tab (as long as he has not clicked on another tab) and display in the content that he cannot access this content. -> you did well already recently for other tabs a few days ago

**Review by smadbe — CHANGES_REQUESTED** [review]

File: src/app/modules/group/pages/group-details/group-details.component.html (hunk 1 above)
Comment (Iloveall) [reply src/app/modules/group/pages/group-details/group-details.component.html]: Fixed

**Review by smadbe — CHANGES_REQUESTED** [review]
If I visit https://dev.algorea.org/branch/feature/display-group-managers-for-members-or-basic-managers/en/#/groups/by-id/689161220395713644;path=52767158366271444/details/managers, I get an error while I would expect the usual message that you are not allowed to see this content (the request should not be done if not a manager nor a member).

File: src/app/modules/group/pages/group-manager/group-manager.component.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+import { Group } from '../../http-services/get-group-by-id.service';
+import { ManagementAdditions } from '../../helpers/group-management';
+
+@Component({
+  selector: 'alg-group-manager',
+  templateUrl: './group-manager.component.html',
+  styleUrls: [ './group-manager.component.scss' ],
+})
+export class GroupManagerComponent {
```
Comment (smadbe) [inline src/app/modules/group/pages/group-manager/group-manager.component.ts:10]: GroupManager**s**Component would have been more appropriate

**Review by smadbe — APPROVED** [review]

File: src/app/modules/group/pages/group-manager/group-manager.component.ts (hunk 2 above)
Comment (Iloveall) [reply src/app/modules/group/pages/group-manager/group-manager.component.ts]: Fixed

**Pull request merged**